extensions/amp-analytics/0.1/test/test-analytics-root.js

```diff
@@ -346,17 +346,18 @@ describes.realWin('AmpdocAnalyticsRoot', {amp: 1}, (env) => {
         toggleExperiment(win, 'visibility-trigger-improvements', false);
       });
 
-      it('should find element and elements by selector', async () => {
+      it('should find AMP element and elements by selector', async () => {
```

Comment:
Is it AMP element?

Reply:
It is an AMP element, but you're right; we don't need this distinction.